media/json-jackson/src/main/java/org/glassfish/jersey/jackson/internal/jackson/jaxrs/util/EndpointAsBeanProperty.java

```diff
@@ -50,7 +50,7 @@ public Std withType(JavaType type) {
         if (_type == type) {
             return this;
         }
-        return new Std(_name, type, _wrapperName, _contextAnnotations, _member, _metadata);
+        return new Std(_name, type, _wrapperName, _member, _metadata);
```

Comment:
The change is due to change in Jackson 2.9.6. API. CQs:
     - Jackson Annotations             (CQ 17606)
     - Jackson Databind                (CQ 17607)
     - Jackson Core                    (CQ 17608)
     - Jackson jaxrs json provider     (CQ 17609)
     - jackson-module-jaxb-annotations (CQ 17610)